kubernetes/reference/adoc/IHV/HPE/SA-RA-CompMod.adoc

```diff
@@ -1,83 +1,66 @@
+include::./SA_vars.adoc[]
 include::./Apollo_vars.adoc[]
 include::./Edgeline_vars.adoc[]
 include::./ProLiant_vars.adoc[]
 include::./Synergy_vars.adoc[]
 
-ifdef::IHV-HPE-Apollo,IHV-HPE-Edgeline,IHV-HPE-ProLiant,IHV-HPE-Synergy[]
-
-NOTE: {an_HPE} https://techlibrary.hpe.com/us/en/enterprise/servers/supportmatrix/suse_linux.aspx[Servers Support & OS Certification Matrices] outlines the minimum version of SLE required for installation, yet later service pack releases may also be used.
-
-include::../../{companyName}/YES.adoc[]
+Even with the broad certification and support model across the range of available {an_HPE} platform models, the following table summarizes which might be a best-practice selection for the various deployment types and focus areas:
 
-Server product-line and model options abound in the https://www.hpe.com/uk/en/servers.html[{an_HPE} portfolio], including:
+.{vn_HPE} Platform Matrix for Deployment Types
+[width="80%",valign="middle",halign="center",options="header"]
+|===
+| System Platform | Baremetal | Hypervisor | Virtual Machine 
 
 ifdef::IHV-HPE-Apollo[]
-include::./Apollo.adoc[]
+| Apollo FixMe | | |
```

Comment:
I guess this should be fixed. :wink:

Reply:
> I guess this should be fixed. wink

as you might have noticed from DC naming, Apollo is not yet included nor is this attribute set, so the "FixMe" is just a placeholder (and BTW, that FixMe is in the Apollo_BOM.adoc ... so again, not an issue yet